rpm/argparse-manpage.spec.tpl

```diff
@@ -48,13 +46,20 @@ BuildRequires: python2-pytest python2-six
 %endif
 %endif
 %endif
+
 %if %{with python3}
 BuildRequires: python3-setuptools python3-devel
 %if %{with check}
 BuildRequires: python3-pytest python3-six
 %endif
 %endif
-%if %{with python3}
+
+%if %{with wheels}
+BuildRequires:  python3-devel
+BuildRequires:  python3-pytest
```

Comment:
A technicality, but this BR might be conditionalized on with_check.

Reply:
BTW you can also require pytest via a "test" extra and than use `%pyproject_buildrequires -r %{?with_check:-x test} `

Reply:
This one is not very nice.  Additional options wouldn't be needed if there was a configuration option in setup.py.

Reply:
What option? Setup.py is not involved directly here at all.

Reply:
There's no option to specify `setup.py test` requirements?
I suppose that direct BuildRequires make this more readable, I think.

Reply:
There is, but it's deprecated together with `setup.py test` command, and not exposed via any suitable PEP 517/518 API.
Considering you maintain the spec upstream, I agree that listing pytest explicitly might be easier, but with a more common downstream/upstream relationship, the spec file is downstream only and hence doing it the proposed way is more future-proof (when upstream adds a new test dependency, the spec file remains unchanged). Feel free to dismiss this suggestion for this particular project thou.